### PR TITLE
koordlet: add more cli flags for audit log and ensure the audit log directory exist

### PR DESCRIPTION
**Pull request by LeoLiuYan**
Signed-off-by: LeoLiuYan <929908264@qq.com>

### Ⅰ. Describe what this PR does

1.  add more cli flags for audit log
2. ensure the audit log directory exist

### Ⅱ. Does this pull request fix one issue?

NONE

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/audit/event_logger.go [hunk 1]
```diff
@@ -212,6 +227,10 @@ func listAuditfiles(dir string) ([]string, error) {
 	pattern := fmt.Sprintf("%s/audit-*.log", dir)
 	files, err := filepath.Glob(pattern)
 	if err == nil {
+		// reverse files
```
Comment (zwzhang0107) [inline pkg/koordlet/audit/event_logger.go:230]: hi, what is the purpose to reverse these files

Comment (LeoLiuYan) [reply pkg/koordlet/audit/event_logger.go:230]: While reading audit files, we expect the audit files order by `date` desc, and the `Glob function` will do the asc by default.   @zwzhang0107

Comment (LeoLiuYan) [reply pkg/koordlet/audit/event_logger.go:230]: Another question, should we add a filter flag in `CLI` to discard any events?

**Comment by LeoLiuYan** [comment]
@zwzhang0107 @jasonliu747 @saintube  Please review the code when you having any idle time.

File: pkg/koordlet/audit/event_logger.go (hunk 1 above)
Comment (zwzhang0107) [reply pkg/koordlet/audit/event_logger.go:230]: now i see the result is already desc, or do you want to get the event from old to new(asc)? maybe add a query param to control this like (desc=true by deafult/false)
image

Comment (LeoLiuYan) [reply pkg/koordlet/audit/event_logger.go:230]: In one file the result is correct, and returned content by `desc` order. But, if we have N files? 

For example, we have those files in the audit log directory: `audit.log`、`audit-2023-01-04-14-28-01.log`、`audit-2023-01-04-13-28-01.log`、`audit-2023-01-04-12-28-01.log`.

 the `Glob function` will return by this order: [`audit-2023-01-04-12-28-01.log`, `audit-2023-01-04-13-28-01.log`, `audit-2023-01-04-14-28-01.log`], when read logger finished `audit.log`, we expect the next file is `audit-2023-01-04-14-28-01.log` which is the latest audit files expect `audit.log`.   @zwzhang0107

Comment (zwzhang0107) [reply pkg/koordlet/audit/event_logger.go:230]: i get your idea, this looks great, good job!!

**Pull request merged**